### DISCUSSION TITLE
Table is too big despite few columns.

### DISCUSSION THREAD

**Discussion by JF-626164** [Power-Components/livewire-powergrid#286] [Q&A]
Hi, so I'm using a version of powergrid without CDNs and only using local files, there are only 9 columns including the checkbox & actions columns. As mentioned in the title, the table is large enough where theres a scroll bar at the bottom of the table letting it run off the side of the screen. 

Is there a way to force it to be wholly on the screen? I.e no scroll?

In the config -> livewire-powergrid.php, I've been using the Bootstrap 5 theme.
 'theme' => \PowerComponents\LivewirePowerGrid\Themes\Bootstrap5::class,

When I swap the theme to use Tailwind, the whole table fits on the page just fine. While that is fixes the 'issue' I prefer the look of the Bootstrap theme.

Any ideas?

Powergrid version: 2.2.6

**Comment by luanfreitasdev** [comment]
I don't think there is a solution for this. This can happen with tailwind too depending on the number of columns.
